### PR TITLE
Fix Illegal Header Warnings

### PR DESCRIPTION
**Pull request by dubeejw**
Closes https://github.com/apache/incubator-openwhisk/issues/2612

### CODE REVIEW

**Comment by dubeejw** [comment]
@rabbah, the configuration setting did not work. Had to filter the headers instead.

**Comment by rabbah** [comment]
Ok. That's probably cleaner anyway. Thanks for knocking this off.

File: tests/dat/actions/base64Web.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function main() {
+	  return {
+		  headers: {
+			  "content-type": "application/json"
+		  },
+		  statusCode: 200,
+		  body: new Buffer(JSON.stringify({'status': 'success'})).toString('base64')
+	  }
+}
```
Comment (markusthoemmes) [inline tests/dat/actions/base64Web.js]: Contains tabs. Why does `scanCode` not catch this @mrutkows ?

File: tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala [hunk 2]
```diff
@@ -788,7 +788,6 @@ trait WebActionsApiTests extends ControllerTestCommon with BeforeAndAfterEach wi
 
                         m(s"$testRoutePath/$path") ~> Route.seal(routes(creds)) ~> check {
                             status should be(OK)
-                            header("content-type").get.toString shouldBe "content-type: application/json"
```
Comment (rabbah) [inline tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala]: Ok - if we have other tests that assert the content type header, do we need the additional integration test?

Comment (dubeejw) [inline tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala]: The other unit tests look fine.

**Comment by dubeejw** [comment]
PG4 691 ⏳ .

File: tests/dat/actions/base64Web.js (hunk 1 above)
Comment (rabbah) [inline tests/dat/actions/base64Web.js]: the tests directory is exempted - chatted with @mrutkows to address.

Comment (mrutkows) [inline tests/dat/actions/base64Web.js]: @markusthoemmes Currently (and historically), the "tests" directory and its subdirs are listed in the "exclusions" section of the default scancode.cfg file.  Messaged Rodric that I would be happy to remove (and want to remove ALL exclusions), but have been waiting for dozens of PRs to merge to allow even considering this.  I will work to see what impact removing the "tests" folder exclusion and see if I can submit a PR.

**Pull request merged**